ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -1765,6 +1765,9 @@ FEDataManager::interpWeighted(const int f_data_idx,
                     dof_id_scratch = F_dof_indices[i];
                     F_dof_map.constrain_element_vector(F_rhs, dof_id_scratch);
                     F_vec.add_vector(F_rhs, dof_id_scratch);
+                    // libMesh sometimes resizes F_rhs inside
+                    // constrain_element_vector
+                    F_rhs.resize(F_dof_indices[0].size());
```

Comment:
Are we relying on `F_rhs` having a consistent size?

Reply:
Good point: we assume, at the moment, that `F_rhs` always has the same size since all components have the same number of dofs, but there is really no reason to assume this. We might as well just resize it to the correct value at the start of each loop.